src/nebari_plugin_self_registration/terraform/modules/self-registration/chart/values.yaml

```diff
@@ -7,7 +7,7 @@ image:
   repository: quay.io/nebari/nebari-self-registration
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: "20240926-1841"
+  tag: "20241011-1937"
```

Comment:
That looks like an old tag (?)

This is what I see in here: https://quay.io/repository/nebari/nebari-self-registration?tab=tags

<img width="1366" alt="Screenshot 2024-10-16 at 9 34 50 pm" src="https://github.com/user-attachments/assets/f7fa8e7a-e0d7-41be-9c17-2125cc4799b8">

We should at some point move to app version tags rather here as well (not necessarily in this PR).